Metro-Records/la-metro-councilmatic
# upgrade to bootstrap 5, enhance mobile display and site accessibility

## Overview

This combines all the previous refactor pr's

Connects #969 

## Testing Instructions

 * Navigate through the site and make sure the styling looks consistent and the functionality still works
